wallet_transactions/wallet_transactions/settings.py

```diff
@@ -12,6 +12,8 @@
 
```

Comment:
`DATABASES = {
    "default": {
        "ENGINE": "django.db.backends.postgresql_psycopg2",
        "NAME": "postgres",
        "USER": "postgres",
        "PASSWORD": "Qwerty512228@",
        "HOST": "127.0.0.1",
        "PORT": "5432",
    }
}`

you need to use .env for these variables

Reply:
create .env file and add settings